src/core/ext/xds/xds_client.h

```diff
@@ -245,6 +243,9 @@ class XdsClient : public DualRefCounted<XdsClient> {
 
     absl::string_view server_uri() const { return server_.server_uri(); }
 
+   protected:
+    void Orphaned() override;
```

Comment:
I think this can just be either public or private, not protected.  There are no subclasses of `XdsChannel`, so I don't want to imply to readers that we expect there to be.

Reply:
we should probably follow up this change with the addition of `final` on every type that I just moved `Orphaned()` to private on.